src/templates/rust-svc/.github/workflows/release.yml

```diff
@@ -27,24 +27,36 @@ jobs:
     concurrency:
       group: release-${{ github.head_ref || 'main' }}
     steps:
+      - name: Set up environment based on branch
+        run: |
+          if [ "$GITHUB_REF_NAME" == "main" ]
+          then
+            echo "PUBLISH_DRY_RUN=0" >> $GITHUB_ENV
+          else
+            echo "CUSTOM_RELEASE_RULES=fix:prerelease,feat:prerelease,chore:prerelease" >> $GITHUB_ENV
```

Comment:
docs:prerelease?

Reply:
docs is already marked as a patch/prerelease by default